src/state/ducks/student/actions.ts

```diff
@@ -4,5 +4,5 @@ import { IStudent, StudentActionTypes } from './types';
 export const fetchStudentData = () =>
   action(StudentActionTypes.FETCH_STUDENTS, [], {
     method: 'get',
-    route: '/students',
+    route: '/StudentForm',
```

Comment:
are you sure you want to use a separate route for this api ?
I think you have not understand the crudcrud.com properly.